config.json

```diff
@@ -24,248 +27,228 @@
+      "difficulty": 5,
+      "slug": "phone-number",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "scrabble-score",
+      "difficulty": 5,
+      "slug": "nucleotide-count",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "largest-series-product",
+      "difficulty": 5,
+      "slug": "word-count",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "roman-numerals",
+      "difficulty": 5,
+      "slug": "robot-name",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "binary",
+      "difficulty": 5,
+      "slug": "allergies",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "raindrops",
+      "difficulty": 5,
+      "slug": "bob",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "allergies",
+      "difficulty": 5,
+      "slug": "pascals-triangle",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "strain",
+      "difficulty": 5,
+      "slug": "bracket-push",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "flatten-array",
+      "difficulty": 5,
+      "slug": "series",
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 5,
       "slug": "atbash-cipher",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "hexadecimal",
+      "difficulty": 6,
+      "slug": "roman-numerals",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "series",
+      "difficulty": 6,
+      "slug": "beer-song",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "sieve",
+      "difficulty": 6,
+      "slug": "etl",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "simple-cipher",
+      "difficulty": 6,
+      "slug": "linked-list",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "luhn",
+      "difficulty": 6,
+      "slug": "grade-school",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "pig-latin",
+      "difficulty": 6,
+      "slug": "binary-search",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "linked-list",
+      "difficulty": 6,
+      "slug": "all-your-base",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "binary-search",
+      "difficulty": 7,
+      "slug": "anagram",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "nth-prime",
+      "difficulty": 8,
+      "slug": "simple-cipher",
       "topics": []
     },
     {
-      "difficulty": 1,
-      "slug": "bracket-push",
-      "topics": []
+      "difficulty": 10,
+      "slug": "react",
+      "topics": [
+        "Reactive programming",
+        "Generics",
+        "Nested classes",
+        "Higher-order functions"
+      ]
     },
     {
       "difficulty": 1,
-      "slug": "pascals-triangle",
+      "slug": "perfect-numbers",
```

Comment:
All subsequent exercises have not yet received difficulty estimates/placements in xjava, hence their positioning here.